kratos/solving_strategies/convergencecriterias/displacement_criteria.h

```diff
@@ -329,6 +345,12 @@ class DisplacementCriteria
     ///@name Protected member Variables
     ///@{
 
+    TDataType mRatioTolerance;      /// The ratio threshold for the norm of the residual
+
+    TDataType mAlwaysConvergedNorm; /// The absolute value threshold for the norm of the residual
+
+    TDataType mReferenceDispNorm;   /// The norm at the beginning of the iterations
```

Comment:
Need to be initialized here or not initialized in the `residual_criteria.h`

Reply:
Okay